Source/Core/Core/HotkeyManager.cpp

```diff
@@ -250,6 +253,7 @@ const std::array<HotkeyGroupInfo, NUM_HOTKEY_GROUPS> groups_info = {
      {_trans("Program Counter"), HK_SHOW_PC, HK_SET_PC},
      {_trans("Breakpoint"), HK_BP_TOGGLE, HK_MBP_ADD},
      {_trans("Wii"), HK_TRIGGER_SYNC_BUTTON, HK_BALANCEBOARD_CONNECT},
+     {_trans("Controller Profile"), HK_NEXT_CONTROLLER_PROFILE, HK_PREV_CONTROLLER_PROFILE },
```

Comment:
The space before the ending brace should be omitted, like the rest of the entries.